src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -717,14 +729,24 @@ async Task<bool> createGlobalConfigFilesAsync()
 
                         try
                         {
+                            // Read shipped file
                             using var fileStream = File.OpenRead(shippedFile);
                             var sourceText = SourceText.From(fileStream);
                             var releaseTrackingData = ReleaseTrackingHelper.ReadReleaseTrackingData(shippedFile, sourceText,
                                 onDuplicateEntryInRelease: (_1, _2, _3, _4, line) => throw new Exception($"Duplicate entry in {shippedFile} at {line.LineNumber}: '{line}'"),
                                 onInvalidEntry: (line, _2, _3, _4) => throw new Exception($"Invalid entry in {shippedFile} at {line.LineNumber}: '{line}'"),
                                 isShippedFile: true);
-                            shippedFilesDataBuilder.Add(releaseTrackingData);
+                            releaseTrackingFilesDataBuilder.Add(releaseTrackingData);
                             versionsBuilder.AddRange(releaseTrackingData.Versions);
+
+                            // Read unshipped file
+                            using var fileStreamUnshipped = File.OpenRead(unshippedFile);
+                            var sourceTextUnshipped = SourceText.From(fileStreamUnshipped);
+                            var releaseTrackingDataUnshipped = ReleaseTrackingHelper.ReadReleaseTrackingData(unshippedFile, sourceTextUnshipped,
+                                onDuplicateEntryInRelease: (_1, _2, _3, _4, line) => throw new Exception($"Duplicate entry in {unshippedFile} at {line.LineNumber}: '{line}'"),
+                                onInvalidEntry: (line, _2, _3, _4) => throw new Exception($"Invalid entry in {unshippedFile} at {line.LineNumber}: '{line}'"),
```

Comment:
I usually prefer to avoid throwing `Exception`. You could maybe use `InvalidOperationException`:

> method call is invalid for the object's current state

Reply:
I see that it's also used (not changed here) a few lines above so I'd recommend updating both places if you go for the change.